### DISCUSSION TITLE
Setting persistent role

### DISCUSSION THREAD

**Discussion by jeanlucthumm** [jackMort/ChatGPT.nvim#290] [Q&A]
Is there a way to set a persistem system message, essentially saving a new custom role for the AI?

Right now I can use `<C-s>` to set the system message, but when I reset the chat using `<C-n>`, the system message is also reset.

This would be useful to set project specific context for every chat.

**Comment by jackMort** [comment]
Not at this point, but that was the idea, don't have time to implement it, but it's on my todo list
